notebooks_tsqr/sources_dashboard.ipynb

```diff
@@ -1,303 +1,50 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    status = Dashboard().report(timeout=0.5)
</pre><p>Why add the dashboard as separate to the NightLog? At least the way I had pictured it, we would first scroll the nightlog, see all the places we are actively able to query, and then below that see the data we have from those sources. </p><p>I am open to differently, but just curious why you chose this way. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Because it cannot work.  The dashboard accesses several different environments (different host servers).  Those other environments are generally not accessible (not authorized) from inside RSP on one environment.

Reply:
<p>I thought that was the point of showing the dashboard of sources, to show <code>oops cannot access xxx-source</code> </p><p>Maybe I am misunderstanding the dashboard as a whole?  </p>

Reply:
You are.  It tells about **access on each server**.   I've had troubles with one endpoint works on one but not the other.  Need to know that earlier.

Reply:
<p>Let's add this topic to the agenda tomorrow?</p>